pkgs/default.nix

```diff
@@ -44,7 +44,6 @@ let
     bootstrap = callPackage ./bootstrap.nix { };
     bootstrapZip = callPackage ./bootstrap-zip.nix { };
     prootTermux = callPackage ./cross-compiling/proot-termux.nix { };
-    qemuAarch64Static = callPackage ./qemu-aarch64-static.nix { };
```

Comment:
Shouldn't the copyright header be adjusted?

BTW: This is always a bit tedious, not sure but maybe https://github.com/fsfe/reuse-tool could help dealing with this stuff in a less distracting way. Didn't you it yet but heard good things about it :)

Reply:
Adjusted.

I think I've evaluated that tool when playing with https://github.com/t184256/python-nix-project-template, but I don't remember why I ended up not using it there. Yeah, the manual updating when we remember to workflow is not ideal, solutions are welcome.